v2/src/lib.rs

```diff
@@ -0,0 +1,837 @@
+//
+// Copyright (c) 2022 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+
+use async_std::task;
+use async_trait::async_trait;
+use base64::{engine::general_purpose::STANDARD as b64_std_engine, Engine};
+use chrono::{NaiveDateTime, Utc};
+use futures::prelude::*;
+use influxdb2::api::buckets::ListBucketsRequest;
+use influxdb2::models::Query;
+use influxdb2::models::{DataPoint, PostBucketRequest};
+use influxdb2::Client;
+use influxdb2::FromDataPoint;
+
+use log::{debug, error, warn};
```

Comment:
As Zenoh coding convention we agreed to never use the `log` module globally, but to prefer explicit calls everywhere (`log::debug!("...")`).
This allows more easy spot and search for logs in code.

Reply:
Done.